lib/rubocop/formatter/formatter_set.rb

```diff
@@ -9,21 +9,21 @@ module Formatter
     # which invoke same method of each formatters.
     class FormatterSet < Array
       BUILTIN_FORMATTERS_FOR_KEYS = {
-        'progress'    => ProgressFormatter,
-        'simple'      => SimpleTextFormatter,
-        'clang'       => ClangStyleFormatter,
-        'fuubar'      => FuubarStyleFormatter,
-        'emacs'       => EmacsStyleFormatter,
-        'json'        => JSONFormatter,
-        'html'        => HTMLFormatter,
-        'files'       => FileListFormatter,
-        'offenses'    => OffenseCountFormatter,
-        'disabled'    => DisabledLinesFormatter,
-        'worst'       => WorstOffendersFormatter,
-        'tap'         => TapFormatter,
-        'quiet'       => QuietFormatter,
-        'autogenconf' => AutoGenConfigFormatter,
-        'pacman'      => PacmanFormatter
+        '[p]rogress'    => ProgressFormatter,
+        '[s]imple'      => SimpleTextFormatter,
+        '[c]lang'       => ClangStyleFormatter,
+        '[fu]ubar'      => FuubarStyleFormatter,
+        '[e]macs'       => EmacsStyleFormatter,
+        '[j]son'        => JSONFormatter,
+        '[h]tml'        => HTMLFormatter,
+        '[fi]les'       => FileListFormatter,
+        '[o]ffenses'    => OffenseCountFormatter,
+        '[d]isabled'    => DisabledLinesFormatter,
+        '[w]orst'       => WorstOffendersFormatter,
+        '[t]ap'         => TapFormatter,
+        '[q]uiet'       => QuietFormatter,
+        '[a]utogenconf' => AutoGenConfigFormatter,
+        '[pa]cman'      => PacmanFormatter
```

Comment:
The abbreviations are repeated here: https://github.com/jfhinchcliffe/rubocop/blob/c29143c5379c14699096ddb114736f8312a64ca6/lib/rubocop/options.rb#L410-L424

Maybe we could DRY this somehow?